test/test.js

```diff
@@ -1,243 +1,306 @@
 // Node
```

Comment:
Was the formatting of this file causing some linting errors? I'm fine with the change but ideally pure refactors should be a separate PR to easily review and merge quick so less people or possibly yourself maybe be affected by conflicts.

Reply:
Make sense, I just ran the prettify command and it changed it 😬